README.md

```diff
@@ -271,7 +271,7 @@ Grype lets you define custom output formats, using [Go templates](https://golang
 
 **Please note:** Templates can access information about the system they are running on, such as environment variables. You should never run untrusted templates.
 
-Here's what the `csv.tmpl` file might look like:
+There are several example templates in the [templates/](https://github.com/anchore/grype/tree/main/templates) directory. Here's what the `(csv.tmpl)[https://github.com/anchore/grype/blob/main/templates/csv.tmpl]` file looks like:
 
 ```gotemplate
```

Comment:
Should we take the example out of the readme and just point users to the templates directory to see examples? If nothing else, it at least eliminates one location to update if the data model ever changes enough that these need to be updated

Reply:
Yeah, that's a good idea, I will update the patch.